web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -58,13 +59,22 @@ export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement =>
     );
   };
 
+  const scrollAccordionToTop = (element: HTMLElement | null, { isDesktop }: { isDesktop: boolean }): void => {
+    if (element) {
+      setTimeout(() => {
+        scrollToTopOfElement(element.parentElement as HTMLDivElement, { isDesktop });
+      }, 200);
```

Comment:
[sand] I know this 200ms timeout is here to account for the time it takes for the menu to open. Currently, this hard-coded value lacks context. It might seem a little overkill, but I think it might be valuable to initialize a variable
```js
const timeForAccordionToOpen = 200  // or some other better name
```
and use that in your `setTimeout` to give a reason for the delay.